src/components/Footer/StyledFooter.js

```diff
@@ -0,0 +1,88 @@
+import styled from 'styled-components'
+import { resolveMedia } from '../../utils/helpers'
+import ContainerWrapper from '../Grid/Container/styledContainer'
+import { Row, Col, Container } from '../Grid'
+import RowWrapper from '../Grid/Row/styledRow';
+
+// Main footer container
+const FooterContainer = styled(Container)`
+
+${resolveMedia.md`
+   
+  `}
+
+`;
+
+
+// CONTACT PANEL
+// Social media contacts row
+const ContactPanelRow = styled(Row)`
+    
+
+`;
+
+// Individual social media contact column
+const ContactPanelColumn = styled(Col)`
+  
+ 
+
+`;
+
+
+
+
+
+// GREY PANEL
+// Row for the grey column under the contact panel
+const GreyPanelRow= styled(Row)`
+
+`;
+
+// Social media counters
+const SocialMediaCounter = styled(Col)`
+
+
+`;
+
+// Footer Link
+const FooterLinks = styled(Col)`
+
+
+`;
+// Blog Button and Language Image 
+const Images = styled(Col)`
+
+
+`;
+// Language Text 
+const LanguageText = styled(Col)`
+   
+    ${resolveMedia.md`
+        display: none;
+    `}
+
+`;
```

Comment:
I think this element should display in all viewports, but as it exists it only displays below the widest size.

Reply:
Sorted!